labelbox/data/serialization/ndjson/label.py

```diff
@@ -95,7 +95,7 @@ def _create_video_annotations(
             if isinstance(
                     annot,
                 (VideoClassificationAnnotation, VideoObjectAnnotation)):
-                video_annotations[annot.feature_schema_id].append(annot)
+                video_annotations[annot.name or annot.schema_id].append(annot)
```

Comment:
I think it likely doesn't actually do anything to this line since we don't use `video_annotations` outside of the purpose of having the values(), but maybe we should make it so that it is `annot.schema_id or annot.name` instead of the reverse?

Reply:
yeah good idea